aten/src/ATen/SparseTensorUtils.h

```diff
@@ -58,6 +58,17 @@ inline Tensor new_values_with_size_of(const Tensor& values, int64_t nnz) {
   return at::empty(size, values.options());
 }
 
+inline Tensor new_values_with_size_of(const Tensor& values, int64_t nnz, ScalarType dtype) {
```

Comment:
Isn't this just `values.new_empty({nnz}, values.options().dtype(dtype))` ?  (I'm not sure about the `.dtype()` call, it might be called ScalarType or something else). Because it's a one-liner I don't think this needs its own helper function.

Reply:
Not necessarily? I think the idea is that `size` could be anything, with other dimensions. It's just a variant of the function above it.

I think this additional variant isn't actually necessary, I'm going to remove it, since I have a `values` tensor with the correct dtype already.

Reply:
Oh, you're right, I didn't realize that `values` had more than one dimension